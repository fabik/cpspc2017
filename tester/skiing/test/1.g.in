189
-618509244 970813770 805983498 -995950048
-1000000000 -882028599 939553751
-1000000000 -86160978 934556088
-1000000000 -786016754 930856941
-1000000000 284908379 926454066
-1000000000 32700034 902364127
-1000000000 -605666710 897548071
-1000000000 99368217 889122117
-1000000000 881754075 889118810
-1000000000 -527000990 888068316
-1000000000 278853888 883511532
-1000000000 133422527 878731234
-1000000000 824031905 877989621
-1000000000 -324139694 871299751
-1000000000 163381161 869827695
-1000000000 -432924277 850202457
-1000000000 -365246721 845278463
-1000000000 -83772122 834862327
-1000000000 937502814 831305842
-1000000000 -523937207 803600757
-1000000000 918956483 770645138
-1000000000 -148383961 770213352
-1000000000 -968929541 763967576
-1000000000 -140508559 731914648
-1000000000 499179512 718979099
-1000000000 -782026601 715807342
-1000000000 542207632 713001658
-1000000000 857490052 711318844
-1000000000 -649831078 694003836
-1000000000 126220565 680351311
-1000000000 -844034670 679303043
-1000000000 -385933174 655872724
-1000000000 538630640 644795901
-1000000000 911923002 641517501
-1000000000 -914365725 637943149
-1000000000 -876326907 627939163
-1000000000 616844362 626782217
-1000000000 905555663 620314884
-1000000000 -928502422 617637645
-1000000000 133351449 602142611
-1000000000 110341135 596965359
-1000000000 -798773161 588594069
-1000000000 -857087741 539592747
-1000000000 311496141 520483989
-1000000000 -687837009 514488299
-1000000000 579243357 507450429
-1000000000 571204478 478032305
-1000000000 -21057330 464149953
-1000000000 148733334 454544246
-1000000000 -708998726 440219787
-1000000000 278851463 428730417
-1000000000 -468333796 415688637
-1000000000 -184908055 409399044
-1000000000 -736293613 394542226
-1000000000 -357539546 388435404
-1000000000 -970121293 381957269
-1000000000 761186264 380706650
-1000000000 -874553904 374309832
-1000000000 714950883 356748268
-1000000000 -897787594 351889748
-1000000000 874658940 343774781
-1000000000 -784662065 330938660
-1000000000 19990137 328349567
-1000000000 980826982 305838604
-1000000000 937662625 296030113
-1000000000 -24296827 271534210
-1000000000 979210461 263740867
-1000000000 257434102 238172451
-1000000000 812170631 225759011
-1000000000 -26400233 206153221
-1000000000 -774030800 201573738
-1000000000 780044767 170358706
-1000000000 -767675439 165922600
-1000000000 463866080 157042711
-1000000000 489916744 144695889
-1000000000 694485059 137846567
-1000000000 -592178521 133564339
-1000000000 955663292 120560883
-1000000000 -470767956 114268259
-1000000000 36726571 75897982
-1000000000 501026368 73814274
-1000000000 -23883406 73627081
-1000000000 239894861 68671662
-1000000000 419763425 65597113
-1000000000 -85021587 44418080
-1000000000 -200595586 43583465
-1000000000 -605417793 41451103
-1000000000 -579189086 38963551
-1000000000 -647480568 27782466
-1000000000 -853205693 17323095
-1000000000 -256536682 5611771
-1000000000 -795247598 -225577
-1000000000 191293844 -18144567
-1000000000 871804249 -20557705
-1000000000 242130157 -27192984
-1000000000 907516299 -29213022
-1000000000 -506427343 -31563964
-1000000000 368617545 -55331195
-1000000000 -737592221 -71959602
-1000000000 -906696114 -76017587
-1000000000 -540611040 -82755989
-1000000000 611041642 -106160481
-1000000000 339736868 -109441643
-1000000000 53490029 -115415172
-1000000000 61753739 -124960936
-1000000000 107304569 -129024107
-1000000000 -370761166 -141346431
-1000000000 -750790309 -142318671
-1000000000 192568405 -148576995
-1000000000 393574916 -176838721
-1000000000 -250359498 -183238648
-1000000000 -381953428 -188150564
-1000000000 609852554 -216204340
-1000000000 -435644191 -223421749
-1000000000 -799826468 -225015151
-1000000000 -362678344 -244878195
-1000000000 270463736 -246212516
-1000000000 101208791 -248441355
-1000000000 852827946 -254727372
-1000000000 -977447355 -255316100
-1000000000 -57341711 -256370407
-1000000000 -987493420 -265988829
-1000000000 -985473722 -279549592
-1000000000 244946151 -279633873
-1000000000 -897174048 -296482817
-1000000000 361888986 -299356582
-1000000000 379803767 -299950402
-1000000000 535871098 -316716510
-1000000000 391728121 -338546409
-1000000000 -123020311 -351305063
-1000000000 -816984952 -373444805
-1000000000 971559885 -374407001
-1000000000 187793089 -404353370
-1000000000 636291482 -420864577
-1000000000 79983846 -426302384
-1000000000 247910557 -435854974
-1000000000 -638686565 -440117485
-1000000000 167151351 -447311225
-1000000000 94745501 -448860411
-1000000000 435450745 -471811897
-1000000000 -509189947 -483218639
-1000000000 490229391 -484133788
-1000000000 -650112618 -515564080
-1000000000 -419634119 -517208153
-1000000000 -580964766 -525831717
-1000000000 186033504 -526142075
-1000000000 -883128896 -526420684
-1000000000 628141159 -526928894
-1000000000 802974159 -530661699
-1000000000 637993065 -538466739
-1000000000 -356627491 -541011761
-1000000000 -506724728 -562849815
-1000000000 718932852 -565127040
-1000000000 380525652 -573609690
-1000000000 645136218 -586045936
-1000000000 731345546 -589292595
-1000000000 53507612 -603563925
-1000000000 446751575 -609320787
-1000000000 407962656 -614585396
-1000000000 -282377410 -624089064
-1000000000 -222829138 -635045417
-1000000000 -998339735 -645727009
-1000000000 -498559032 -668980848
-1000000000 771318252 -670737009
-1000000000 -238186546 -680592004
-1000000000 54952520 -690075195
-1000000000 867827032 -691498425
-1000000000 -870032354 -700367407
-1000000000 -218733491 -702009841
-1000000000 -879466023 -713606852
-1000000000 899081789 -716254457
-1000000000 855056835 -748674583
-1000000000 -338870407 -774646567
-1000000000 928596488 -778910506
-1000000000 -98462041 -783300812
-1000000000 468702232 -783372011
-1000000000 842583010 -785702085
-1000000000 717974978 -829492154
-1000000000 -607128241 -829991836
-1000000000 -631465155 -854180867
-1000000000 -661203969 -877847000
-1000000000 -572549437 -882026016
-1000000000 -455672750 -897247030
-1000000000 86090783 -899480651
-1000000000 640523600 -915612844
-1000000000 44704324 -960838746
-1000000000 -368883624 -963579523
-1000000000 -787709469 -971247558
-1000000000 641761054 -973899634
-1000000000 -345953944 -986386184
