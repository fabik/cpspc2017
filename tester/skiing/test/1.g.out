7
-618509244 970813770
-882028599 939553751
-968929541 763967576
-998339735 -645727009
-787709469 -971247558
-345953944 -986386184
805983498 -995950048
